bundles/org.openhab.binding.snmp/src/main/java/org/openhab/binding/snmp/internal/SnmpTargetHandler.java

```diff
@@ -208,7 +209,17 @@ public void processPdu(@Nullable CommandResponderEvent event) {
         final PDU pdu = event.getPDU();
         final String address = ((UdpAddress) event.getPeerAddress()).getInetAddress().getHostAddress();
         final String community = new String(event.getSecurityName());
-
+        
+        if ((pdu.getType() == PDU.V1TRAP) && config.community.equals(community) && (pdu instanceof PDUv1)){
```

Comment:
Is the formatter really not putting a space here?
```suggestion
        if ((pdu.getType() == PDU.V1TRAP) && config.community.equals(community) && (pdu instanceof PDUv1)) {
```

Reply:
Strange, but spotless finds problems only in xml files. It should be fine now